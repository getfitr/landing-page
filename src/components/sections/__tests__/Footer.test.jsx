import renderer from 'react-test-renderer';
import Footer from '../Footer';

test('Shows the Footer', () => {
  const footer = renderer.create(<Footer />);
  const tree = footer.toJSON();
  expect(tree).toMatchSnapshot();
});