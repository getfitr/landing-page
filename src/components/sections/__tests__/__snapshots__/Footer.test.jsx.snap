// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shows the Footer 1`] = `
<div>
  This is the Footer
</div>
`;
